app/templates/gentelella/users/events/_table.html

```diff
@@ -72,15 +72,15 @@
                         {% if flag %}
                             {% for id, item in ticket_stats.iteritems() %}
                                 <div>
-                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span> 
+                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span>
                                 </div>
                                 <br>
                              {% endfor %}
                         {% else %}
                             {{ _("No Ticket Information") }}
                         {% endif %}
                     </td>
-                    <td><a href="{{ url }}" target="_blank">{{ url }}</a></td>
+                    <td style="width:200px;"><a href="{{ url }}" target="_blank">{{ url }}</a></td>
```

Comment:
Why are you looking inside `jquery.dataTables.min.js` and `jquery.dataTables.js` ? They are library files ... The column widths must be specified by *you* in a separate js file